open-metadata-implementation/access-services/data-platform/data-platform-api/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/properties/DeployedDatabaseSchema.java

```diff
@@ -0,0 +1,61 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.dataplatform.properties;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.List;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.PUBLIC_ONLY;
+
+/**
+ * The type DeployedDatabaseSchema is an asset in Apache Cassandra Database.
+ */
+@JsonAutoDetect(getterVisibility = PUBLIC_ONLY, setterVisibility = PUBLIC_ONLY, fieldVisibility = NONE)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DeployedDatabaseSchema extends Source {
+
+    private String name;
+    private String displayName;
+    private List<TabularSchema> tabularSchemaList;
```

Comment:
What is the meaning of tabularSchemaList in deployedDatabaseSchema?

Reply:
For asset Deployed Database Schema it can attache with several different schema elements with it. Such as for Cassandra keyspace is a deployed database schema and when it is created it can attach 0 to N Cassandra tables as tabular schemas.